.travis.yml

```
@@ -16,6 +16,8 @@ matrix:
       env: TOXENV=py38
     - python: 3.9-dev
       env: TOXENV=py39
+    - python: pypy3
+      env: TOXENV=pypy3 PYPY_VERSION=3.5-v7.0.0
 
 install: pip install -U wheel tox codecov
```

This should be enough to make `PYPY_VERSION` work:

```suggestion
install:
  - |
      if [[ ! -z "$PYPY_VERSION" ]]; then
        export PYPY_VERSION="pypy$PYPY_VERSION-linux64"
        wget "https://bitbucket.org/pypy/pypy/downloads/${PYPY_VERSION}.tar.bz2"
        tar -jxf ${PYPY_VERSION}.tar.bz2
        virtualenv --python="$PYPY_VERSION/bin/pypy3" "$HOME/virtualenvs/$PYPY_VERSION"
        source "$HOME/virtualenvs/$PYPY_VERSION/bin/activate"
      fi
  - pip install -U wheel tox codecov
```